docs/content/content/multilingual.md

```diff
@@ -0,0 +1,151 @@
+---
+date: 2016-01-02T21:21:00Z
+menu:
+  main:
+    parent: content
+next: /content/example
+prev: /content/summaries
+title: Multilingual Mode
+weight: 68
+toc: true
+---
+Hugo supports multiple languages side-by-side (added in `Hugo 0.17`). Define the available languages in a `Languages` section in your top-level `config.yaml` (or equivalent).
+
+Example:
+
+```
+DefaultContentLanguage = "en"
```

Comment:
Actually I was referring to only line 17. That line is TOML while line 18-32 is YAML